sdk/servicebus/azure-servicebus/tests/test_partitioned_sessions.py

```diff
@@ -698,6 +698,7 @@ def test_qsession_get_set_state_with_receiver(live_servicebus_config, partitione
 
 @pytest.mark.liveTest
 def test_qsession_by_servicebus_client_list_sessions_with_receiver(live_servicebus_config, partitioned_session_queue):
+    pytest.skip("TODO: Investigation with the service team on the list_sessions api")
```

Comment:
Please create an issue to track it, with some information/stacktrace of what you see that you don't like and mention it here

Reply:
I have to chuckle at least a little bit:  (From the original PR)  https://github.com/Azure/azure-sdk-for-python/pull/4961/files#r328365665

(To save you a click; you aren't the first one asking this question, I'll dig to find the answer.)